UIforETW/Utility.cpp

```diff
@@ -431,108 +859,116 @@ bool Is64BitWindows()
 #else
 	// http://blogs.msdn.com/b/oldnewthing/archive/2005/02/01/364563.aspx
 	BOOL f64 = FALSE;
-	bool bIsWin64 = IsWow64Process(GetCurrentProcess(), &f64) && f64;
+	const bool bIsWin64 = ( ::IsWow64Process(::GetCurrentProcess(), &f64) && f64 );

```

Comment:
The outer set of parentheses seem odd - there's no precedence for them to clarify.